docker/boot_notebook.bash

```diff
@@ -54,6 +54,18 @@ cat > .jupyter_config.json <<EOF
 }
 EOF
 
+# cookie security options for cross domain
+cat > "$HOME/.jupyter/jupyter_notebook_config.py" <<EOF
+c.JupyterHub.tornado_settings = {
```

Comment:
`c.JupyterHub.tornado_settings` is repeated ... I guess is a bad typo?

Reply:
It is not repeated, it is different.